### ISSUE TITLE
Add current value field to RWA document model

### ISSUE THREAD

**Issue by ryanwolhuter** [powerhouse-inc/connect#462]
The current value field is a special one, because it depends on the current time. Since documents are stored as static json, this cannot be saved directly in the document. 

Both the front and back ends need to be able to show this value, which needs to be calculated at the time of request. To keep this consistent, we will expose a `calculateCurrentValue` utility function in the document model, which can then be imported and used by both.

**Comment by CallmeT-ty** [comment]
@ryanwolhuter This issue is completed isn't it?

**Comment by CallmeT-ty** [comment]
@ryanwolhuter Is the current value suppose to be present in the API for example? or is that another one of those derived fields the clients should calculate?

**Comment by ryanwolhuter** [comment]
@CallmeT-ty afaik we can close this one as done 👍 we have only implemented this to work with Treasury Bill assets (and we can expand it to work on more) but from what I got from Teep, this is enough for now.

The current value is available in the API. This is indeed a derived field. It is a special case because this value _cannot_ be added to the actual document model at all, because this value can only be computed when you pass in the current time. This is why instead of having an operation like ADD_CURRENT_VALUE or similar, we instead provide a utility function that does this calculation in either the front or back end. 

In the case of switchboard, it imports the code for doing this calculation from the `utils` of the document model in document model libs and then runs that function to calculate the current value at the time that the api gets called. It then adds this data to the data in the api response, so that from the user's perspective the "current value" is indeed in the response from switchboard the same as if it was in the document model.

Similar with the frontend, we import the same exact function in connect and use it to calculate the value for current value that we show for each asset in the assets table. Again this looks like the value is part of the document model, but in fact it is being calculated dynamically.
